tests/test_velocity_verlet_integrator.py

```diff
@@ -1,3 +1,7 @@
+import jax
+
+jax.config.update("jax_enable_x64", True)
```

Comment:
is this new? (not sure if we just want to stuff this in a `conftest` later)

Reply:
Yeah, will fix with the conftest update, put here for now so the tests pass